.gitignore

```diff
@@ -31,3 +31,5 @@ log/
 .redcar/
 /nbproject
 .idea/
+
+.ruby-version
```

Comment:
is there meant to not be a trailing newline here?

Reply:
We didn't write any of this.

Reply:
oh, duh! then ignore my comment!